docs/content/docs/build/data-modeling.md

```diff
@@ -39,15 +45,18 @@ const cell = fluidContainer.initialObjects["custom-cell"];
 
 ## Dynamic objects
 
-A shared object can be created dynamically by the container at runtime. Dynamic objects are both created and loaded dynamically and are always stored as references within another shared object. In other words, a container can create an object dynamically, and you must store references to those objects within another shared object.
+A Fluid object can be created by the container at runtime. **Dynamic objects** are both created and loaded
+dynamically. When your code can creates an object dynamically, it must store a reference to the object within another Fluid object so that your code can later
```

Comment:
```suggestion
dynamically. When your code creates an object dynamically, it must store a reference to the object within another Fluid object so that your code can later
```